doc-content/shared-kie-docs/src/main/asciidoc/Install/proc-standalone-editors.adoc

```diff
@@ -0,0 +1,160 @@
+[id="proc-standalone-editors_{context}"]
+= Configuring the {PRODUCT} standalone editors
+
+{PRODUCT} provides standalone editors that are distributed in a self-contained library providing an all-in-one JavaScript file for each editor. The JavaScript file uses a comprehensive API to set and control the editor.
+
+You can install the standalone editors in three ways:
+
+ifdef::DROOLS,JBPM,OP[]
+* Use hosted JavaScript files directly
+endif::[]
+* Download each JavaScript file manually
+* Use the NPM package
+
+.Procedure
+. Install the standalone editors using one of the following methods:
++
+--
+ifdef::DROOLS,JBPM,OP[]
+*Use hosted JavaScript files directly*: For this method, add the following `<script>` tags to your HTML page:
+
+.Script tags for your HTML page
+[source]
+----
+<script src="https://<YOUR_PAGE>/dmn/index.js"></script>
+<script src="https://<YOUR_PAGE>/bpmn/index.js"></script>
+----
+endif::[]
+
+*Download each JavaScript file manually*: For this method, follow these steps:
+
+.. Download the JavaScript files.
+.. Add the downloaded Javascript files to your hosted application.
+.. Add the following `<script>` tag to your HTML page:
++
+.Script tag for your HTML page for the DMN editor
+[source]
+----
+<script src="https://<YOUR_PAGE>/dmn/index.js"></script>
+----
++
+.Script tag for your HTML page for the BPMN editor
+[source]
+----
+<script src="https://<YOUR_PAGE>/bpmn/index.js"></script>
+----
+
+*Use the NPM package*: For this method, follow these steps:
+
+.. Go to the https://npm.registry.redhat.com/[NPM registry] and download the NPM package.
```

Comment:
this step is not needed, remove it please.

Reply:
Done